### PR TITLE
Run pre-commit on everything.

### PR DESCRIPTION
**Pull request by RyanFleck**
- Formats all documents
- Organizes imports

### CODE REVIEW

**Review by yozohu — COMMENTED** [review]
looks fine, but if linting passes is this necessary?
cus everyones formatter might be different. i think passing lint should be enough

File: bellboy/actors/elevator.py [hunk 1]
```diff
@@ -16,8 +20,9 @@
 
 def buttonHovered(depth_deque: deque):
     """
-    rough implementation of a buttonHovered method. processes data deque.
-    tells us which button was hovered.
+    rough implementation of a buttonHovered method.
+
```
Comment (yozohu) [inline bellboy/actors/elevator.py:24]: why all this spacing 😞

**Review by yozohu — COMMENTED** [review]
looks good but hcekc comments

**Review by yozohu — APPROVED** [review]

File: bellboy/actors/elevator.py (hunk 1 above)
Comment (RyanFleck) [reply bellboy/actors/elevator.py:24]: Basically with docstrings, the first line is the 'summary', and anything after that is the comment.

so... standards?

If you'd like everything to be on the same line you can use commas in the future.

**Pull request merged**